Calculator/Parser/Extensions/Character.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Character.swift
+//  Calculator
+//
+//  Created by Gleb Kovalenko on 17.07.2022.
+//
+
+// MARK - Character
+
+extension Character {
+    var isDot: Bool {
+        self == Character(".")
+    }
+    
+    /// Checked if it is a prefix unary operation character, and if so, returns prefix unary operation token, otherwise nill
```

Comment:
Комментарии не пишутся в прошедшем времени, только настоящее.

```suggestion
    /// Check if it is a prefix unary operation character and returns prefix unary operation token if it's exists
```